src/pages/Login/Login.js

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect } from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import styled from 'styled-components';
+import qs from 'qs';
+import {
+  restApiKey,
+  redirectUrl,
+  clientSecret,
+  kakaoAuthUrl,
+} from '../../config';
+
+const Login = () => {
+  const [searchParams] = useSearchParams();
+  const code = searchParams.get('code');
+  const navigate = useNavigate();
+
+  const body = qs.stringify({
+    grant_type: 'authorization_code',
+    client_id: restApiKey,
+    redirect_uri: redirectUrl,
+    code: code,
+    client_secret: clientSecret,
+  });
+
+  const getToken = async () => {
+    try {
+      await fetch('https://kauth.kakao.com/oauth/token', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/x-www-form-urlencoded;charset=utf-8',
+        },
+        body,
+      })
+        .then(res => res.json())
+        .then(data => {
+          localStorage.setItem('kakao_token', data.access_token);
+          sendToken();
+        });
+    } catch (err) {
```

Comment:
```suggestion
        .then(res => res.json())
        .then(data => {
          sendToken(data.access_token);
        });
    } catch (err) {
```
카카오 토큰은 백엔드한테 보내는 용도로만 사용하고 있으니 로컬 스토리지에 저장 후 사용하는 방법도 있지만
이렇게 바로 인자로 넘겨줘도 됩니다. 확인 후 적용해주세요!
만약 이렇게 바뀐다면 remove 부분도 같이 수정이 필요합니다

Reply:
@we-dohyeon 도현님, 도현님이 적어주신대로 수정해서 시도를 해봤는데 토큰을 받아오지를 못합니다.